src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -389,52 +397,160 @@ sub-<label>/[ses-<label>/]
+diffusion images will require splitting the DWI scheme in several runs.
+Because researchers will generally optimize the data splits, these will likely
+not be directly concatenable anymore.
+BIDS permits defining arbitrary groupings of these multipart scans with the
+following metadata:
+
+| **Key name**    | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                      |
+| --------------- | --------------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| MultipartID     | REQUIRED              | [string][]    | A unique (per participant) label tagging DWI runs that are part of a multipart scan.                                                                                                                                 |
+
+JSON example:
+
+```JSON
+{
+  "MultipartID": "dwi_1"
+}
+```
+
+For instance, if there are two phase-encoding directions (`AP`, `PA`), and
+two runs each, and the intent of the researcher is that all of them are
+part of a unique multipart scan, then they will tag all four runs with the
+same `MultipartID` (shown at the right-hand side of the file listing):
+
+```Text
+sub-<label>/[ses-<label>/]         # MultipartID
+  dwi/
+    sub-1_dir-AP_run-1_dwi.nii.gz  # dwi_1
+    sub-1_dir-AP_run-2_dwi.nii.gz  # dwi_1
+    sub-1_dir-PA_run-1_dwi.nii.gz  # dwi_1
+    sub-1_dir-PA_run-2_dwi.nii.gz  # dwi_1
+```
+
+If, conversely, the resarcher wanted to store two multipart scans, one possibility
+is to combine matching phase-encoding directions:
+
+```Text
+sub-<label>/[ses-<label>/]         # MultipartID
+  dwi/
+    sub-1_dir-AP_run-1_dwi.nii.gz  # dwi_1
+    sub-1_dir-AP_run-2_dwi.nii.gz  # dwi_1
+    sub-1_dir-PA_run-1_dwi.nii.gz  # dwi_2
+    sub-1_dir-PA_run-2_dwi.nii.gz  # dwi_2
+```
+
+Alternatively, the researcher's intent could be combining opposed phase-encoding
+runs instead:
+
+```Text
+sub-<label>/[ses-<label>/]         # MultipartID
+  dwi/
+    sub-1_dir-AP_run-1_dwi.nii.gz  # dwi_1
+    sub-1_dir-AP_run-2_dwi.nii.gz  # dwi_2
+    sub-1_dir-PA_run-1_dwi.nii.gz  # dwi_1
+    sub-1_dir-PA_run-2_dwi.nii.gz  # dwi_2
+```
+
+The `MultipartID` metadata MAY be used with the
+[`acq-<label>`](../99-appendices/09-entities.md#acq) key/value pair, for example:
+
+```Text
+sub-<label>/[ses-<label>/]             # MultipartID
+  dwi/
+    sub-1_acq-shell1_run-1_dwi.nii.gz  # dwi_1
+    sub-1_acq-shell1_run-2_dwi.nii.gz  # dwi_2
+    sub-1_acq-shell2_run-1_dwi.nii.gz  # dwi_1
+    sub-1_acq-shell2_run-2_dwi.nii.gz  # dwi_2
+```
+
+### Other RECOMMENDED metadata
 
+At minimum, the `PhaseEncodingDirection` and `TotalReadoutTime` metadata
```

Comment:
"Minimum" + "RECOMMENDED" feels like a mismatch.

```suggestion
The `PhaseEncodingDirection` and `TotalReadoutTime` metadata
```

Reply:
This was meant to say that amongst all the metadata entries we could recommend, these two are really necessary if one wants to be able to perform certain processing (just in case someone wants to recommend something in regards, not just removing the awkward combo).